### ISSUE TITLE
weight_decay

### ISSUE THREAD

**Issue by YIBO-0805** [Yunfan-Li/Contrastive-Clustering#22]
Why the parameter 'weight_decay'  set in code is 0.

**Comment by mark375chen** [comment]
> We didn't tune that hyper-parameter during the experiments. Perhaps a proper weight decay could further improve the performance.

What might be an appropriate weight decay for CIFAR10?

**Comment by Yunfan-Li** [comment]
We didn't tune that hyper-parameter in our experiments. You may try the commonly used 1e-3 or 1e-4.
